guide/07-working-with-bigdata/07-use_proximity.ipynb

```diff
@@ -47,7 +47,7 @@
     "      <td><a href=\"https://enterprise.arcgis.com/en/portal/latest/use/geoanalytics-trace-proximity-events.htm\"><p align=\"left\">trace_proximity_events</p></a></td>\n",
```

Comment:
<p>All the text in the <strong>Trace Proximity Events </strong>section displays as bold, including the paragraph text and the code text within the cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Looks great now!